master/src/ComputerMonitoringWidget.cpp

```diff
@@ -293,19 +294,21 @@ void ComputerMonitoringWidget::runMousePressAndHoldFeature( )
 		selectedInterfaces.first()->hasValidFramebuffer() )
    {
 		m_ignoreMousePressAndHoldEvent = true;
+		m_ignoreNumberOfMouseEvents = 3;
```

Comment:
Please define a `static constexpr auto IgnoredNumberOfMouseEventsWhileHold = 3` in ComputerMonitoringWidget.h to eliminate this anonymous value in the code.